angular/src/app/project-form/project-form.component.ts

```diff
@@ -26,6 +27,7 @@ export class ProjectFormComponent implements OnInit {
         this.router.navigate(['/discover']);
       },
       error => {
+        this.snackBar.open("One or more of the fields are filled in incorrectly.", "Dismiss");
         console.log(this.projectForm.value);
```

Comment:
console log should only be used for error messages as of now. Shouldn't be used to log value of other attributes. Might even consider not using `console.log` for errors later as well because we're using `snackbars`.